src/home/components/HomePage/HomePage.tsx

```diff
@@ -84,18 +57,13 @@ const HomePage: React.FC<HomePageProps> = props => {
           <RequirePermissions
             requiredPermissions={[PermissionEnum.MANAGE_ORDERS]}
           >
-            <div className={classes.cardContainer}>
-              <HomeAnalyticsCard
-                title={"Sales"}
-                testId="sales-analytics"
-                icon={
-                  <Sales
-                    className={classes.icon}
-                    fontSize={"inherit"}
-                    viewBox="0 0 64 64"
-                  />
-                }
-              >
+            <Box
+              display="grid"
+              __gridTemplateColumns="1fr 1fr"
+              gap={8}
+              marginBottom={8}
+            >
+              <HomeAnalyticsCard title={"Sales"} testId="sales-analytics">
```

Comment:
**issue:** this string is untranslated. Not blocking because it was like that before, but it would be good to fix this.